### ISSUE TITLE
I this project still maintained? 

### ISSUE THREAD

**Issue by BobLoblaw411** [javinizer/Javinizer#397]
I was wondering if this project is still maintained? 

jvlflame hasn´t contributed for over 12month now, he might have abandoned this project and it seems there are more and more features that won´t work, especially Javlibrary is  a pain to use since the upped the Cloudflare protection.

Has anyone identified a different project that helps managing their JAV liberary?

I would really REALLY love Javinizer to live on, unfortunately I´m not able to contribute anything myself.

**Comment by RabidWolf** [comment]
For Javinizer r18.dev scraper still works but it does miss some things out. Javlibrary is hit and miss for me but at the min it's not working as they changed cf_chl_2 to cf_chl_3 so it no longer works.

**Comment by nothing2obvi** [comment]
I'm wondering the same thing. Seems like I used Javinizer not that long ago but it's not working anymore because of the `cf_chl_3` change.

**Comment by nothing2obvi** [comment]
@BobLoblaw411 Find any alternatives?

**Comment by gummykage** [comment]
Yes this is still maintained, many hotfixes, what's working: AVEntertainment, DMM, JAV321, Javlibrary (join discord), mgstagejp, tokyohots, and r18.dev.

**Comment by nothing2obvi** [comment]
@BobLoblaw411 @RabidWolf 

I was able to join the Discord and get help thanks to @jvlflame. Now Javinizer works :)

**Comment by RabidWolf** [comment]
> @BobLoblaw411 @RabidWolf
> 
> I was able to join the Discord and get help thanks to @jvlflame. Now Javinizer works :)

Is this for Javlibrary? If so can you explain how to fix it please.

**Comment by nothing2obvi** [comment]
@RabidWolf 

You can find the link to their Discord on [this page](https://javinizer.gitbook.io/docs/troubleshooting/frequently-asked-questions). The instructions are in the `flaresolverr` channel (I think "flaresolverr work around")
